README.md

```diff
@@ -351,6 +351,18 @@ ______________________________________________________________________
 
 ______________________________________________________________________
 
+## Lightning Lite
```

Comment:
Remember this needs @williamFalcon approval :)

Reply:
Yes, have pinged him. Won't merge until then.